### PR TITLE
Fix #1425: Add card to my submissions

### LINKED ISSUE #1425: Bug in mysubmission page
Card for my submissions is not visible. Check screenshot for more info.
screen shot 2017-12-26 at 01 12 20

Make submission card visible.

### PR DESCRIPTION
**Pull request by NSDCars5**
Fixes #1425.

Fixed screenshot:

127 0 0 1_8888_web_challenges_challenge-page_1_my-submission laptop

### CODE REVIEW

File: frontend/src/css/modules/my-challenge-all-submission.scss [hunk 1]
```diff
@@ -7,3 +7,15 @@
 .all-submission-table-scroll {
 	overflow-x: scroll;
 }
+
+.all-submission-card {
```
Comment (aka-jain) [inline frontend/src/css/modules/my-challenge-all-submission.scss]: Reuse the classes for this card from other elements. No need to add a new class.

**Comment by NSDCars5** [comment]
@aka-jain Made the required changes.

**Comment by aka-jain** [comment]
Please paste the updated screenshot.

**Comment by NSDCars5** [comment]
Screenshot helped me catch a small problem. :) Attached below.

127 0 0 1_8888_web_challenges_challenge-page_1_my-submission laptop 1

File: frontend/src/views/web/challenge/my-submission.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <section ng-if="challenge.isParticipated" class="ev-sm-container ev-view ">
     <div class="row">
         <div class="col s12 m12">
-            <div class=" ev-card-panel ev-z-depth-5 ">
+            <div class="ev-challenge-view ev-card-panel ev-z-depth-5 ">
```
Comment (aka-jain) [inline frontend/src/views/web/challenge/my-submission.html]: I don't think we need this class here? Please check again @NSDCars5

File: frontend/src/views/web/challenge/my-submission.html [hunk 3]
```diff
@@ -53,7 +53,7 @@
                         </div>
                     </div>
                 </div>
-                <div class=" ev-card-body exist-team-card">
+                <div class="ev-challenge-view ev-card-panel exist-team-card">
```
Comment (aka-jain) [inline frontend/src/views/web/challenge/my-submission.html]: Don't remove `ev-card-body` from here.
Don't add `ev-challenge-view` here.
Do these changes and then paste a screenshot of result please
@NSDCars5

File: frontend/src/views/web/challenge/my-submission.html (hunk 2 above)
Comment (NSDCars5) [reply frontend/src/views/web/challenge/my-submission.html]: The class was providing the bottom margin to separate the two cards.

With the class:

localhost_8888_web_challenges_challenge-page_1_my-submission laptop

Without it:

localhost_8888_web_challenges_challenge-page_1_my-submission laptop 1

Should I remove it and let the cards come together? This same class is used in the top cards of all other tabs where there is more than one card.

**Comment by NSDCars5** [comment]
@aka-jain Hey, so it seems that there was an extra closing tag that was making them into different cards, otherwise they were meant to be part of the same card. Confirmed this by checking the same page on the host side.

Host page:

localhost_8888_web_challenges_challenge-page_1_my-challenge-all-submission laptop

Fixed participant page:

localhost_8888_web_challenges_challenge-page_1_my-submission laptop 2

Apologies for the confusion.

File: frontend/src/views/web/challenge/my-submission.html (hunk 3 above)
Comment (NSDCars5) [reply frontend/src/views/web/challenge/my-submission.html]: @aka-jain Made some more changes.

**Pull request merged**